app/models/history.rb

```diff
@@ -147,6 +149,14 @@ def self.report_reminder(patient: nil, created_by: 'Sara Alert System', comment:
     create_history(patient, created_by, HISTORY_TYPES[:report_reminder], comment) unless patient&.preferred_contact_method.nil?
   end
 
+  def self.vaccination(patient: nil, created_by: 'Sara Alert System', comment: 'User added a new vacciation to the monitoree record..')
+    create_history(patient, created_by, HISTORY_TYPES[:vaccination], comment)
+  end
+
+  def self.vaccination_edit(patient: nil, created_by: 'Sara Alert System', comment: 'User edited a vacciation on the monitoree record.')
```

Comment:
thoughts on making the language for comments between different data types consistent? either by adding more detail to the history messages for the other data types like you did here or making the vaccination messages the same format as the existing history messages?

I'm also thinking that it wouldn't hurt to try to keep the history messages more concise if possible since they probably add up a lot in terms of space in production

Reply:
Agreed they should be consistent. Updated here: https://github.com/SaraAlert/SaraAlert/pull/721/commits/1af7e86f6a26f91f7b2b6563d00d493acf16da44#diff-08422754c6b565c51a7e094488199ec370790128736cd382aa486715d3126452R154